nats-top.go

```diff
@@ -36,12 +36,15 @@ var (
 	caCertOpt     = flag.String("cacert", "", "Root CA cert")
 	skipVerifyOpt = flag.Bool("k", false, "Skip verifying server certificate")
 
+	// Nats context options
+	contextName = flag.String("c", "", "The name of the context to use. If not specified, the currently selected context is used.")
```

Comment:
would prefer here `--context` like the nats cli too
```suggestion
	contextName = flag.String("context", "", "The name of the context to use. If not specified, the currently selected context is used.")
```